clang/lib/Analysis/FlowSensitive/ASTOps.cpp

```diff
@@ -281,6 +282,17 @@ ReferencedDecls getReferencedDecls(const FunctionDecl &FD) {
   Visitor.TraverseStmt(FD.getBody());
   if (const auto *CtorDecl = dyn_cast<CXXConstructorDecl>(&FD))
     Visitor.traverseConstructorInits(CtorDecl);
+  if (const auto *Method = dyn_cast<CXXMethodDecl>(&FD);
```

Comment:
I think it would help to ~copy the comment from the PR description to here. It would also be good to explain why we are more liberal with captured parameters.

Reply:
Expanded on the commit description here, but let me know if this got too verbose.

Reply:
No, I think its great! Makes things much easier if we ever decide/need to change it.